README.md

```diff
@@ -411,6 +411,34 @@ Name | Type | Required | Description | Example
 elementId ("element" prior to Appium v 1.22) | string | no | Unique identifier of the element to send the keys to. If unset then keys are sent to the current application under test. | 21045BC8-013C-43BD-9B1E-4C6DC7AB0744
 keys | array | yes | Array of keys to type. Each item could either be a string, that represents a key itself (see the official documentation on XCUIElement's [typeKey:modifierFlags: method](https://developer.apple.com/documentation/xctest/xcuielement/1500604-typekey?language=objc) and on [XCUIKeyboardKey constants](https://developer.apple.com/documentation/xctest/xcuikeyboardkey?language=objc)) or a dictionary with `key` and `modifierFlags` entries, if the key should also be entered with modifiers. | ['h', 'i'] or [{key: 'h', modifierFlags: 1 << 1}, {key: 'i', modifierFlags: 1 << 2}] or ['XCUIKeyboardKeyEscape'] |
 
+!!! note
```

Comment:
Should be:

```
> **note**
>
> xxxx
```

syntax like the `Note` in the top of this README?
![Screenshot 2024-01-09 at 11 23 41 AM](https://github.com/appium/appium-mac2-driver/assets/5511591/9a70f820-966e-4589-a84e-359adcd24d41)

Reply:
Just synchronized to what @eglitise proposed earlier. I am not really concerned about what formatting to use there